illumos-utils/src/zfs.rs

```diff
@@ -344,26 +412,24 @@ impl Zfs {
         datasets: &[String],
     ) -> Result<Vec<DatasetProperties>, anyhow::Error> {
         let mut command = std::process::Command::new(ZFS);
-        let cmd = command.args(&["list", "-d", "1", "-rHpo"]);
+        let cmd = command.args(&[
+            "get",
+            "-d",
+            "1",
+            "-rHpo",
```

Comment:
Two questions:

* `-d 1` gives the properties for the dataset in question and any of its children. Is that intentional, or should we drop both that and `-r`?
* If it is intentional, can we add a comment explaining why and drop the `-r`? The man page says `-d N` makes it recursive up to the given depth.

Reply:
Good point - I think the `-r` is not necessary. I'll add docs to the function itself.